src/aspire/basis/dirac.py

```diff
@@ -18,9 +18,12 @@ def __init__(self, sz, mask=None, dtype=np.float32):
         """
         Initialize an object for Dirac basis
         :param sz: The shape of the vectors for which to define the basis.
+        May be a 2-tuple or an integer, in which case, a square basis is assumed.
         :param mask: A boolean _mask of size sz indicating which coordinates
             to include in the basis (default np.full(sz, True)).
         """
+        if isinstance(sz, int):
+            sz = (sz, sz)
```

Comment:
Every other basis uses `size` instead of `sz`. Should we change this for consistency?